bin/infrasim-init

```diff
@@ -11,8 +11,8 @@ from infrasim.ipmi import get_ipmi
 from infrasim.qemu import get_qemu
 import netifaces
 
-INFRASIM_TEMP_CONF = "/usr/local/etc/infrasim/conf/infrasim.yml"
-INFRASIM_CONF = "/etc/infrasim/infrasim.yml"
+INFRASIM_TEMP_CONF = os.environ['HOME'] + "/.infrasim/template/infrasim.yml"
```

Comment:
we can eliminate usage of template in folder of ~/.infrasim, because all templates should appear in /usr/local/infrasim/template. No need to copy them from /usr/local/infrasim/template to ~/.infrasim/template neither.